### PR TITLE
clientMutationId -> clientSubscriptionId

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: examples/todo/js/subscriptions/AddTodoSubscription.js [hunk 1]
```diff
@@ -12,11 +12,8 @@ export default class AddTodoSubscription extends RelaySubscriptions.Subscription
   getSubscription() {
     return Relay.QL`subscription {
       addTodoSubscription {
-        clientMutationId
         todoEdge {
-          __typename
           node {
-            __typename
```
Comment (edvinerikson) [inline examples/todo/js/subscriptions/AddTodoSubscription.js:19]: did you try to run the example after this change?

Comment (taion) [inline examples/todo/js/subscriptions/AddTodoSubscription.js:19]: Yes, though I messed up the signature on base62. One sec.

**Comment by taion** [comment]
Dropped `base62`. Using `.toString(36)` instead. (base62 wasn't actually used here).

File: examples/todo/js/subscriptions/AddTodoSubscription.js (hunk 1 above)
Comment (taion) [inline examples/todo/js/subscriptions/AddTodoSubscription.js:19]: It seems to work. There's a flash of the same edge being present twice, but that's there against master as well.

File: src/SubscriptionRequest.js [hunk 2]
```diff
@@ -81,13 +81,13 @@ export default class SubscriptionRequest {
     }
   }
 
-  setDisposable(dispoable: Subscription): void {
+  setDisposable(disposable: Subscription): void {
```
Comment (edvinerikson) [inline src/SubscriptionRequest.js:84]: awesomeness

File: examples/todo/js/subscriptions/AddTodoSubscription.js (hunk 1 above)
Comment (edvinerikson) [inline examples/todo/js/subscriptions/AddTodoSubscription.js:19]: Okay :+1: The reason I had this was because the babel plugin didn't add this automatically for some reason.

**Comment by edvinerikson** [comment]
Is this ready to be merged?

File: examples/todo/js/subscriptions/AddTodoSubscription.js (hunk 1 above)
Comment (taion) [inline examples/todo/js/subscriptions/AddTodoSubscription.js:19]: Is it needed? It's not in the transpiled output for `AddTodoMutation` either.

Comment (edvinerikson) [inline examples/todo/js/subscriptions/AddTodoSubscription.js:19]: Hmm maybe it is Relay that adds it then..? Relay couldn't get the type of the node so it didn't know how to cache it or something

**Comment by taion** [comment]
More or less. My todos here (HAH!) are:
1. Update the example to determine the topic on the backend (using graphql-relay-subscription?)
2. Move the subscription logic to some `RelaySubscriptionsEnvironment` that extends `RelayEnvironment`

I can make those separate PRs or add them as commits to this PR.

**Comment by edvinerikson** [comment]
Thanks! :+1:

File: examples/todo/js/subscriptions/AddTodoSubscription.js (hunk 1 above)
Comment (taion) [inline examples/todo/js/subscriptions/AddTodoSubscription.js:19]: Wait a sec... okay, it was a console warning. Hmm.

Comment (edvinerikson) [inline examples/todo/js/subscriptions/AddTodoSubscription.js:19]: when the babel plugin source code is back (https://github.com/facebook/relay/issues/1348) we can try to see if we can fix it there.

Just send another PR re-adding them for now :D

Comment (taion) [inline examples/todo/js/subscriptions/AddTodoSubscription.js:19]: Ah, okay, Relay does it for mutations only in https://github.com/facebook/relay/pull/657.
